### PR TITLE
Add scvelo

### LINKED ISSUE #42: Add scvelo

### PR DESCRIPTION
**Pull request by ivirshup**
@WeilerP, I'd like to include scvelo on our new ecosystem listing (check out the [pr](https://github.com/scverse/scverse.github.io/pull/70) and [preview](https://deploy-preview-70--jade-cajeta-1bcca0.netlify.app/projects).

Does this all look good to you?

--------

Name of the tool: scvelo

Short description: 

> scVelo is a scalable toolkit for RNA velocity analysis in single cells, based on Bergen et al. [(Nature Biotech, 2020)](https://doi.org/10.1038/s41587-020-0591-3).

How does the package use scverse data structures (please describe in a few sentences): Uses AnnData

-   [x] The code is publicly available under an [OSI-approved](https://opensource.org/licenses/alphabetical) license
-   [x] The package provides versioned releases
-   [x] The package can be installed from a standard registry (e.g. PyPI, conda-forge, bioconda)
-   [x] The package uses automated software tests and continuous integration
-   [x] The package provides API documentation via a website or README
-   [x] The package uses scverse datastructures where appropriate (i.e.anndata/mudata and their modality-specific extensions)
-   [x] I am the author or maintainer of the tool and agree on listing the package on the scverse website

Fixes #42

### CODE REVIEW

File: packages/scvelo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: scvelo
```
Comment (WeilerP) [inline packages/scvelo/meta.yaml:1]: Is this just an ID? Otherwise, I'd change it to _scVelo_ as in the description.

**Review by WeilerP — APPROVED** [review]
@ivirshup, besides that one comment, everything looks good to me!

File: packages/scvelo/meta.yaml (hunk 1 above)
Comment (ivirshup) [reply packages/scvelo/meta.yaml:1]: I think so, let's find out.

```suggestion
name: scVelo
```

**Comment by ivirshup** [comment]
Great, thanks for taking a look!

**Pull request merged**